libcudacxx/test/libcxx/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+
+set(LIBCXX_LIT_VARIANT "libcxx" CACHE STRING
+    "Configuration variant to use for LIT.")
+
+set(LIBCXX_TEST_LINKER_FLAGS "" CACHE STRING
+    "Additonal linker flags to pass when compiling the tests")
+
+# <nv/target> is found in the parent project, it is required in all headers now
+set(LIBCXX_TEST_COMPILER_FLAGS
+    "-I${libcudacxx_SOURCE_DIR}/include" CACHE STRING
+    "Additonal compiler flags to pass when compiling the tests")
+
+# sccache cannot handle the -Fd option generationg pdb files
+if (MSVC)
+  set(CMAKE_MSVC_DEBUG_INFORMATION_FORMAT Embedded)
+endif()
+
+# Ensure we catch all warnings
+if (MSVC)
+  set(LIBCUDACXX_TEST_COMPILER_FLAGS "${LIBCUDACXX_TEST_COMPILER_FLAGS} /W4 /WX")
```

Comment:
Suggestion: This syntax is fine, but I wanted to point out [`string(APPEND ...)`](https://cmake.org/cmake/help/latest/command/string.html#append) -- it's a bit more compact.

Reply:
Also, is this supposed to be `LIBCUDACXX` or `LIBCXX`?